Formula/tmux.rb

```diff
@@ -67,6 +67,9 @@ def caveats
     <<~EOS
       Example configuration has been installed to:
         #{opt_pkgshare}
+
+      On macOS, add the following to etc/tmux.conf or ~/.tmux.conf
+        set -g default-terminal screen-256color
```

Comment:
I don't mean that we should suggest adding the path `etc/tmux.conf` to the caveat -- I meant that we should consider creating a file `tmux.conf` in `$HOMEBREW_PREFIX/etc` (which is the path given by the variable `etc`) that contains only
```tmux
set -g default-terminal screen-256color
```
(but only on macOS).

I was wondering what the reasons to prefer one approach over the other would be.

Reply:
The only reason to prefer the caveat is that I don't know how to create and populate a file correctly in homebrewese. I've re-read the Formula Cookbook and I don't see an example of creating a file like this. I know how to do it using shell and redirection, but I feel like that is not the right way to do it. I'm trying to pick through https://rubydoc.brew.sh/ to see if I can identify a good option. I check a bunch of Formula at random and didn't see any that do something like this.

Reply:
```rb
(etc/"file").write <<~EOS 
  sometext
EOS
```